CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+Acetate is open source, and contributions are welcome!
+
+## Installation
+
+1. Fork and clone the repo
+2. `cd acetate && npm intall`
+
+This will set up everything you need to develop and test Acetate.
+
+## Code Style
+
+Code style and conventions are automatically checked when you run `npm test` with [semistandard](https://www.npmjs.com/package/semistandard).
+
+## Testing
+
+Tests are written with [tape](https://www.npmjs.com/package/tape) and run with [tap-spec](https://www.npmjs.com/package/faucet).
```

Comment:
@paulcpederson looks like tests are not just run with tape, but with istanbul as well.

```js
  "scripts": {
    "prepublish": "npm test",
    "test": "npm run lint && npm run test:unit && npm run test:features && istanbul report",
    "test:unit": "istanbul cover ./node_modules/tape/bin/tape --dir ./coverage/unit -- test/*.js | tap-spec",
    "test:features": "istanbul cover ./node_modules/tape/bin/tape --dir ./coverage/features -- test/features/*/test.js | tap-spec",
    "test:dev": "tape test/features/*/test.js | tap-spec",
    "lint": "semistandard"
  }
```

Also you're linking to faucet (not used) and referencing tap-spec as the test runner -- tap-spec doesn't run the tests, it just formats tape output to look like mocha's spec reporter.